jquery.simulate.js

```diff
@@ -200,8 +200,8 @@ $.extend( $.simulate.prototype, {
 		return event;
 	},
 
-	dispatchEvent: function( elem, type, event ) {
-		if ( elem[ type ] ) {
+	dispatchEvent: function( elem, type, event, options ) {
+		if ( elem[ type ] && !options ) {
```

Comment:
Ah, realized I misunderstood @dmethvin's comment.  Seems like it's really just for click events. I'm happy to make the change re: feature detection, but am not sure what that looks like. Will look into that a bit and update the PR.